pkg/webui/components/news-panel/news-item/news-item.styl

```diff
@@ -0,0 +1,40 @@
+// Copyright © 2023 The Things Network Foundation, The Things Industries B.V.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+.item
+  transition: background-color $ad.m ease-in-out
+  border-radius: $br.xl
+
+  &:hover
+    cursor: pointer
```

Comment:
Instead of recreating the link behavior here, you should wrap the whole component around an `<AnchorLink />`.